Shared/mods/deathmatch/logic/CLodModels.cpp

```diff
@@ -0,0 +1,512 @@
+        {3501, 3500},   {8583, 8731},   {8585, 8733},   {8689, 8934},   {8687, 8718},   {8677, 8752},   {8675, 8728},   {3613, 3728},   {3629, 3672},
+        {8671, 8769},   {3661, 3662},   {3677, 3681},   {8663, 8704},   {8659, 8998},   {8657, 8997},   {3741, 3742},   {8655, 8762},   {8653, 8994},
+        {8651, 8995},   {8649, 8986},   {3821, 3832},   {8647, 8988},   {8645, 8984},   {8643, 8726},   {8639, 8734},   {8637, 8772},   {8635, 8795},
+        {8633, 8802},   {8631, 8800},   {8629, 8796},   {3981, 4031},   {3997, 4045},   {4013, 4065},   {4029, 4136},   {16178, 16464}, {16242, 16544},
+        {8611, 8917},   {8609, 8919},   {4122, 4066},   {4154, 4155},   {4186, 4057},   {8435, 8727},   {4250, 4385},   {4282, 4416},   {4314, 4448},
+        {4346, 4531},   {17508, 17737}, {17636, 17722}, {8883, 8963},   {8947, 8958},   {8581, 8714},   {4538, 4539},   {4570, 4578},   {4602, 4580},
+        {9267, 9477},   {4666, 4667},   {8575, 8741},   {4730, 4758},   {9523, 9535},   {9587, 9621},   {8567, 8716},   {4858, 4975},   {4890, 4946},
+        {8565, 8723},   {9907, 9935},   {8555, 8709},   {10035, 9996},  {8547, 8742},   {8545, 8738},   {5114, 5254},   {5146, 5236},   {5178, 5227},
+        {10419, 10581}, {10483, 10545}, {5274, 5286},   {10611, 10662}, {8535, 8970},   {8533, 8817},   {5402, 5578},   {5434, 5599},   {10931, 11221},
+        {5498, 5571},   {8529, 8774},   {11123, 11192}, {8525, 8902},   {5626, 5664},   {8523, 8900},   {8521, 8830},   {5722, 5963},   {5754, 5909},
+        {5786, 5945},   {5818, 6003},   {8519, 8897},   {5882, 5884},   {8517, 8898},   {8515, 8754},   {8513, 8820},   {6010, 5858},   {6042, 6084},
+        {8511, 8761},   {8509, 8925},   {6138, 6140},   {8507, 8822},   {16613, 16614}, {6234, 6241},   {8503, 8819},   {6298, 6463},   {6330, 6377},
+        {8501, 9075},   {8499, 8929},   {12851, 13234}, {12915, 13052}, {6490, 6491},   {6522, 6523},   {13107, 13115}, {13171, 13384}, {13235, 13238},
+        {8483, 8782},   {13363, 13366}, {8465, 8744},   {13491, 13492}, {8417, 8812},   {8405, 8813},   {13683, 13859}, {6874, 7149},   {8395, 8701},
+        {8355, 8349},   {8351, 8347},   {8335, 8334},   {7034, 7352},   {8315, 8318},   {7098, 7152},   {8311, 8312},   {8283, 8284},   {8281, 8282},
+        {8255, 8159},   {8253, 8252},   {8249, 8248},   {8245, 8025},   {8219, 8222},   {8217, 8223},   {8215, 8226},   {7450, 7814},   {7482, 7771},
+        {7514, 7680},   {7546, 7732},   {8213, 8224},   {8201, 8239},   {5511, 5557},   {5503, 5534}
+});
+
+
+std::uint32_t CLodModels::GetObjectLODOfModel(std::uint32_t objectID)
+{
+    const auto it = LOD_MODELS->find(objectID);
+    if (it != LOD_MODELS->end())
+        return it->second;
+    return 0;
+}
+
+std::uint32_t CLodModels::GetObjectModelOfLOD(std::uint32_t lodID)
+{
+    for (const auto& [objectID, lodModel] : *LOD_MODELS)
```

Comment:
`LOD_MODELS.get()` instead of `*` for better readability maybe

Reply:
this is fine in terms of readability.